src/components/todoList.css

```diff
@@ -0,0 +1,22 @@
+ul {
+  list-style: none;
+}
+
+li {
+  display: flex;
```

Comment:
- Awesome work so far 👍 . I think it would be more Awesome if you make a separate class for the `display: flex` property and use it in the JSX. Since that would make your code more DRY!

    ```css
    /*  todoList.css */
    display: flex;
    ```

    ```jsx
    // todoItem.js
    <li className="flex"></li>
    ```